File: loan_manager/loan_manager/report/user_transactions_report/user_transactions_report.js
frappe.query_reports["User Transactions Report"] = {


    
    "formatter": function(value, row, column, data, default_formatter) {
        value = default_formatter(value, row, column, data);

        if (column.fieldname === "amount") {
            if (data.amount > 0) {
                return `<span style="color: green; font-weight: bold;">${value}</span>`;
            } else if (data.amount < 0) {
                return `<span style="color: red; font-weight: bold;">${value}</span>`;
            }
        }
        return value;
    }
};

File: loan_manager/loan_manager/report/user_transactions_report/user_transactions_report.py
import frappe
from frappe import _



def formatter(value, row, column, data, default_formatter):
    if column.get("fieldname") == "amount":
        if value > 0:
            return f'<span style="color: green;">{value}</span>'
        elif value < 0:
            return f'<span style="color: red;">{value}</span>'
    return default_formatter(value, row, column, data)
